src/redux/actions/group-actions.js

```diff
@@ -22,7 +22,24 @@ export const fetchGroup = (apiProps) => ({
 
 export const addGroup = (groupData) => ({
   type: ActionTypes.ADD_GROUP,
-  payload: GroupHelper.addGroup(groupData),
+  payload: GroupHelper.addGroup(groupData).catch((err) => {
```

Comment:
Couldn't this be axios plugin?

Reply:
Its gonna have to change. This event should replace the shole wizard step. The notification won't suffice. I got my hands on this: https://marvelapp.com/prototype/237ah31g/screen/71426702

Reply:
Mhh it could be an plugin, but meh.

Reply:
So it probably won't be possible. This is a very specific scenario. Sometimes we need to prevent the error notification and consume the outcome in the wizard. You can see the code. Now we can add an interceptor that would create a message format that will not kill the UI, but that is probably for a different PR.